templates/web/base/reports/_list-filters.html

```diff
@@ -1,10 +1,14 @@
 [% select_status = BLOCK %]
     <select class="form-control js-multiple" name="status" id="statuses" multiple
-      data-all="[% loc('All') %]" data-all-options='["open","closed","fixed"]'
-    [%~ IF c.cobrand.on_map_default_status == 'open' %]
-      data-none="[% loc('Unfixed reports') %]"
+    data-all="[% loc('All') %]"
+    [% IF c.user_exists AND c.user.is_superuser OR c.user.belongs_to_body(body.id) %]
+    [% options = []; FOR group IN filter_states; FOR state IN group.1; NEXT IF state == 'hidden'; options.push(state); END; END %]
+    data-all-options='["[% options.join('", "') %]"]'
+    [%~ ELSE ~%]
+    data-all-options='["open","closed","fixed"]'
```

Comment:
This comment is still outstanding - fixed should only be included if `has_fixed_state`, same as printing out the options.